__tests__/integration/multisig.test.js

```diff
@@ -658,22 +654,6 @@ describe('send tx (HTR)', () => {
             txId: expect.any(String),
             value: 2,
           },
-          {
```

Comment:
This is the tx hex affected by the change in the lib, now that we are using the best utxo selection algorithm.

Previously, this would randomly fail depending on the utxo selected. Now we will always select the same (the best)